python/docs/_quarto.yml

```diff
@@ -50,9 +50,6 @@ website:
       - guide/execution.qmd
   - title: Examples
     contents: examples
-  - title: Community
```

Comment:
Should we keep the community page? Or is this just removing the side nav for now? I could see a case where we either have a separate "Contributor Guide" or put that under the community page. Something like https://pydata-sphinx-theme.readthedocs.io/en/stable/community/index.html. It could be a place for documentation about how things work internally / how to contribute to them.

Reply:
this only removes the side nav.  I think it looks funny having a single page with one item on the side nav.

I think adding a contributor guide would be great. we can add back the side bar on this page when we want to add more pages under it.